evaluate.py

```diff
@@ -731,9 +686,12 @@ def __init__(self, shape_dict, layout="NCHW", preproc=None):
 
         self.inputs = {name : image}
 
-    def Validate(self, m, ref_outputs=[]):
+    def Validate(self, m, ref_outputs=[], data=[]):
```

Comment:
We have discussed with you, that `profile` can be moved after inference and validation. In this case we shouldn't run model in Validate function.